plugins/filter_kubernetes/kube_meta.c

```diff
@@ -87,14 +89,149 @@ static int file_to_buffer(const char *path,
     return 0;
 }
 
+/* Run command to get Kubernetes authorization token */
+static int get_token_with_command(const char *command, 
+                                  char **out_buf, size_t *out_size)
+{
+
+#ifdef FLB_HAVE_KUBE_TOKEN_COMMAND
```

Comment:
it's very weird to have this inside the function. Can it be around the function? And then also you have a check around the declaration and usage of the function.